knowledge-base/export-a-report-from-telerik-report-server-via-dot-net-client.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Export a report from Telerik Report Server via .NET Client
+description: Render a report to another format from Telerik Report Server via .NET Client.
+type: how-to
+page_title: Download a report from Telerik Report Server via .NET Client
+slug: export-a-report-from-telerik-report-server-via-dot-net-client
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Products</td>
+			<td>Progress® Telerik® Reporting, Report Server</td>
+		</tr>
+	</tbody>
+</table>
+    
+## Description
+
+[Telerik Report Server](https://www.telerik.com/report-server) has its REST API exposed, allowing you to consume resources from third-party applications as an administrator, after logging in. You can find *examples* with JavaScript client in the REST API section of the online documentation:  
+
+- [Login](../../report-server/implementer-guide/apis/rest-api/examples/javascript/login) and saving the access token for further usage;
+- [Resources](../../report-server/implementer-guide/apis/rest-api/examples/javascript/get-resources) - listing resources like reports available on the Report Server;
+- [Export](../../report-server/implementer-guide/apis/rest-api/examples/javascript/export) a report by getting its ID first. 
+  
+## Solution
+
+The same JavaScript client can be written as a .NET client that executes requests and handles their response in code:  
```

Comment:
There is no information about the following code. What it does, why it is needed, how and where to copy/paste it?